### PR TITLE
Make WDist.ToString() return <cells>c<units>

### PR DESCRIPTION
**Pull request by atlimit8**

### CODE REVIEW

**Comment by penev92** [comment]
I like the idea. Is this used for anything other than documentation?

File: OpenRA.Game/WDist.cs [hunk 1]
```diff
@@ -103,7 +103,7 @@ public int CompareTo(object obj)
 
 		public int CompareTo(WDist other) { return Length.CompareTo(other.Length); }
 
-		public override string ToString() { return Length.ToString(); }
+		public override string ToString() { return (Length / 1024).ToString() + "c" + (Length % 1024).ToString(); }
```
Comment (pchote) [inline OpenRA.Game/WDist.cs]: You need to be careful about negative numbers, this won't work as-is.

**Comment by atlimit8** [comment]
Fixed subcell sign.

File: OpenRA.Game/WDist.cs [hunk 2]
```diff
@@ -103,7 +103,7 @@ public int CompareTo(object obj)
 
 		public int CompareTo(WDist other) { return Length.CompareTo(other.Length); }
 
-		public override string ToString() { return Length.ToString(); }
+		public override string ToString() { return (Length / 1024).ToString() + "c" + (Math.Abs(Length) % 1024).ToString(); }
```
Comment (pchote) [inline OpenRA.Game/WDist.cs]: I suspect this still won't work for negative numbers less than 1024.

**Comment by atlimit8** [comment]
Fixed sign for [-1023, -1].

**Comment by Mailaender** [comment]
> ### NukePower
> Requires trait: [`BodyOrientation`](#bodyorientation).
> <table>
> <tr><th>Property</th><th>Default Value</th><th>Type</th><th>Description</th></tr>
> <tr><td>FlightVelocity</td><td>0c512</td><td>1D World Distance</td><td>Visual ascent velocity in WDist / tick </td></tr>
> </table>

Works as promised. :white_check_mark:

**Comment by abcdefg30** [comment]
Looks good to me, too. :+1: / :white_check_mark:

**Pull request merged**